examples/custom_datatype/server_types_custom.c

```diff
@@ -10,6 +10,8 @@
 
 #include "custom_datatype.h"
 
+#define AUTOCLEANUP_FLAG "cleanup"
```

Comment:
I don't get why this requires a dedicated test.
It is not particularly useful to know that memory is lost if nobody cleans up.
Allowing memleaks here would rather hide memleaks in other places.